streams/src/main/java/org/apache/kafka/streams/state/internals/RocksDBStore.java

```diff
@@ -46,6 +48,16 @@
 import java.util.NoSuchElementException;
 import java.util.Set;
 
+/**
+ * A persistent key-value store based on RocksDB.
+ *
+ * Note that array typed keys may result in incorrect caching behavior.
```

Comment:
I'd add: "If you are working on byte arrays, for example, you may want to wrap them with the `Bytes` class."